src/Checkbox.jsx

```diff
@@ -29,15 +29,12 @@ export default class Checkbox extends React.Component {
   };
   constructor(props) {
     super(props);
-    let checked = false;
-    if ('checked' in props) {
-      checked = props.checked;
-    } else {
-      checked = props.defaultChecked;
-    }
+
+    const checked = 'checked' in props ? props.checked : props.defaultChecked;
+
     this.state = {
       checked,
-      focus: false,
+      focus: false
```

Comment:
This make lint broken: https://travis-ci.org/react-component/checkbox/jobs/205669775